h2o-py/docs/metrics.rst

```diff
@@ -9,6 +9,12 @@ Metrics in H2O
   :show-inheritance:
   :noindex:
 
+.. automodule:: h2o.model.metrics
+  :members:
+  :undoc-members:
+  :show-inheritance:
+  :noindex:
+
```

Comment:
after some refactoring, specialized metrics where moved from `h2o.model.metrics_base` module to dedicated `h2o.model.metrics.xxx`